lib/inspec/dependencies/dependency_set.rb

```diff
@@ -14,9 +14,9 @@ class DependencySet
     # @param cwd [String] Current working directory for relative path includes
     # @param vendor_path [String] Path to the vendor directory
     #
-    def self.from_lockfile(lockfile, cwd, cache, backend, opts = {})
+    def self.from_lockfile(lockfile, cwd, cache, backend, opts = {}, parent_profile = nil)
```

Comment:
Placing a positional arg after the hash arg is unrubyish - usually the last literals are swept into an implicit hash.  I'm not sure if / how this would work.  Can `parent_profile` appear before `opts`, or be made one of the opts?

Reply:
Yeah, my original stab at this put it in options but thats really for target options and gets pulled into the profile as such. It felt ugly putting stuff in there for transport and then removing it later as it does not belong. Do we feel ok just putting this new field at the beginning of the arg list?